### PR TITLE
Add press release about cybersecurity awareness month

### PR DESCRIPTION
**Pull request by maennchen**
- [x] I acknowledge my contribution to the website does not assert comparative or superlative differences of one product, project, company or individual over another.
- [x] In discussion at the security WG / will have to be cleared with marketing WG before publish. See https://the-eef.slack.com/archives/CTP7P1E9X/p1729537530135769
- [ ] Wait for GO on publication 

# Screenshots

Screenshot from 2024-10-22 21-35-20
image

### CODE REVIEW

**Comment by maennchen** [comment]
Waiting for approval from Alistair. I'll take it out of draft status so that you can merge without me being around.

File: priv/posts/marketing/20241021190832_cybersecurity_awareness_month.md
Comment (starbelly) [inline priv/posts/marketing/20241021190832_cybersecurity_awareness_month.md:2]:
```suggestion
  "title": "Cybersecurity Awareness Month (CAM 2024)",
```

**Review by starbelly — APPROVED** [review]

**Pull request merged**
